db/agents.yml

```diff
@@ -507,6 +507,12 @@ agents:
     os: Android
     examples:
       - PodcastRepublic/18.0 (Linux; U; Android 8.0.0;dreamqltesq/R16NW)
+  - regex: '^Podcoin'
+    name: Podcoin
+    type: Mobile App
+    os: null
+    examples:
+      - Podcoin
```

Comment:
I'm am seeing a few other user agents in my logs over the past couple weeks:

| Row | remote_agent                                | count |
|-----|---------------------------------------------|-------|
| 1   | Podcoin                                     | 24863 |
| 2   | Podcoin/123 CFNetwork/978.0.7 Darwin/18.6.0 | 55    |
| 3   | Podcoin/168 CFNetwork/978.0.7 Darwin/18.6.0 | 26    |
| 4   | Podcoin/168 CFNetwork/976 Darwin/18.2.0     | 5     |
| 5   | Podcoin/168 CFNetwork/808.3 Darwin/16.3.0   | 3     |
| 6   | Podcoin/146 CFNetwork/978.0.7 Darwin/18.6.0 | 1     |
| 7   | Podcoin/60 CFNetwork/978.0.7 Darwin/18.6.0  | 1     |
| 8   | Podcoin/146 CFNetwork/976 Darwin/18.2.0     | 1     |

I guess those are older app versions?  Very small amount of the traffic, and they do get picked up by this regex, so I'm not too worried about.  If the % ticks up, can add another matcher for `os: iOS`.